core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -280,6 +284,11 @@ private[spark] class AppStatusListener(
   private def updateBlackListStatus(execId: String, blacklisted: Boolean): Unit = {
     liveExecutors.get(execId).foreach { exec =>
       exec.isBlacklisted = blacklisted
+      if (blacklisted) {
+        appStatusSource.foreach(_.BLACKLISTED_EXECUTORS.inc(1))
```

Comment:
You could just say `.inc()` - 1 is the default. Same everywhere else.

Reply:
ok